lib/modules/auth/widgets/form_header.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+import 'package:podcasty/values/app_theme.dart';
+
+class FormHeader extends StatelessWidget {
+  final String headerText;
+  final bool isLeftOriented;
+
+  const FormHeader({
+    Key? key,
```

Comment:
You can write super.key directly here as to avoid calling super constructor.

Reply:
Resolved